Examples/Io/Root/include/ActsExamples/Io/Root/RootPropagationStepsWriter.hpp

```diff
@@ -75,6 +75,7 @@ class RootPropagationStepsWriter
   std::vector<int> m_layerID;      ///< layer identifier if
   std::vector<int> m_approachID;   ///< surface identifier
   std::vector<int> m_sensitiveID;  ///< surface identifier
+  std::vector<int> m_material;     ///< flag material if present
```

Comment:
it's been a long time since I wrote root files - it doesn't like bools so you have to use int?

Reply:
I think so, on the other hand, we can use this to flag the type of material (not done yet):
- 0 ... no material
- 1 ... homogenous
- 2 ... binned
- 3 ... proto material